### DISCUSSION TITLE
Killcurly how will the admin notice that i have logged out of my account.

### DISCUSSION THREAD

**Discussion by pengwdong** [3kh0/ext-remover#603] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Killcurly how will the admin notice that i have logged out of my account.

**Comment by cooltoo37** [comment, marked as answer]
telepathic message is sent to them by voldemort and it tells them that you have signed out duh

**Reply by MilkMaker23** [reply]
killcurry is not working i did the sign out and restarted but everythig is still blocked

**Reply by Aka-but-nice** [reply]
> telepathic message is sent to them by voldemort and it tells them that you have signed out duh
Dude is literally Woke, he is so Woke, he's asleep

**Reply by Aka-but-nice** [reply]
> killcurry is not working i did the sign out and restarted but everythig is still blocked

What extension do you have?
And it's Killcurly, not Killcurry
